design-proposals/preferences.md

```diff
@@ -0,0 +1,584 @@
+# Overview
+
+A common pattern for IaaS is to have an abstraction separating the sizing and performance characteristics of a virtual machine from the user defined values related to launching their custom application. This pattern is evident across all the major cloud providers as well as open source IaaS projects like OpenStack. AWS calls this abstraction an [instance type](https://aws.amazon.com/ec2/instance-types/), GCP calls it a [machine type](https://cloud.google.com/compute/docs/machine-types#custom_machine_types). Azure calls it an [instance VM size](https://docs.microsoft.com/en-us/azure/virtual-machines/sizes), and OpenStack has [flavors](https://docs.openstack.org/nova/latest/user/flavors.html) and [image properties](https://docs.openstack.org/glance/latest/admin/useful-image-properties.html).
+
```

Comment:
What is the pendant of googles machine families and azures or aws instance types in the presented approach, and what how is the sizing in the approach modeled?

Reply:
I can add some examples if that would help?